lib/compress/zstd_compress.c

```diff
@@ -2462,6 +2478,16 @@ static int ZSTD_disableLiteralsCompression(const ZSTD_CCtx_params* cctxParams)
     }
 }
 
+static int ZSTD_limitMaxCBlockSize(const ZSTD_CCtx_params* cctxParams)
```

Comment:
minor : comment the role of this function, and the meaning of its return values.